src/UserManager.ts

```diff
@@ -263,7 +263,11 @@ export class UserManager {
     }
 
     protected async _useRefreshToken(state: RefreshState): Promise<User> {
-        const response = await this._client.useRefreshToken({ state });
+        const response = await this._client.useRefreshToken({
+            state,
+            timeoutInSeconds: this.settings.silentRequestTimeoutInSeconds
+                ? this.settings.silentRequestTimeoutInSeconds : 10,
```

Comment:
With #339 `this.settings.silentRequestTimeoutInSeconds` should always have a defined value and we can avoid hardcoding this `10` value.